integration-tests/src/test/java/org/web3j/protocol/scenarios/FastRawTransactionManagerIT.java

```diff
@@ -36,17 +33,15 @@
 import org.web3j.tx.response.QueuingTransactionReceiptProcessor;
 import org.web3j.utils.Convert;
 
-import static junit.framework.TestCase.assertFalse;
-import static junit.framework.TestCase.assertTrue;
+import static org.junit.jupiter.api.Assertions.assertFalse;
+import static org.junit.jupiter.api.Assertions.assertTrue;
 import static org.web3j.tx.TransactionManager.DEFAULT_POLLING_ATTEMPTS_PER_TX_HASH;
 
 public class FastRawTransactionManagerIT extends Scenario {
 
     private static final int COUNT = 10; // don't set too high if using a real Ethereum network
     private static final long POLLING_FREQUENCY = 15000;
 
-    @Rule public TestRule benchmarkRun = new BenchmarkRule();
```

Comment:
?

Reply:
@BenchmarkOptions(concurrency = 2, warmupRounds = 0, benchmarkRounds = 5)
This would be the equivalent of that rule and can be used at the class level. Would like some opinions on the values that should be passed here.
[Docs about that here.](https://shantonusarker.blogspot.com/2017/01/junit-benchmark.html)

Reply:
Why this benchmark is used for, concurrency issues or actual benchmarking? 

I think concurrency and warmups are ok, maybe we don't need 5 rounds to have an approximate idea of the test execution time. IMO 2-3 would suffice but I'm a total newbie in benchmarking.